_proto/hapi/chart/metadata.proto

```diff
@@ -90,4 +90,7 @@ message Metadata {
 
 	// KubeVersion is a SemVer constraint specifying the version of Kubernetes required.
         string kubeVersion = 17;
+
+	// Whether or not the app version can be override by flag --app-version (during install or update)
+	bool overrideAppVersion = 18;
```

Comment:
please change this to `overrideMetadata`; there may be more fields in the future that the user may wish to override, and I don't see a reason why we need to create `override` fields for each field we want to allow the user to override.